kroxylicious-bom/pom.xml

```
@@ -100,14 +100,14 @@
 
             <dependency>
                 <groupId>io.kroxylicious</groupId>
-                <artifactId>kroxylicious</artifactId>
+                <artifactId>kroxylicious-runtime</artifactId>
                 <version>${project.version}</version>
             </dependency>
 
             <!-- Optional Production dependencies -->
             <dependency>
-                <groupId>io.kroxylicious</groupId>
-                <artifactId>kroxylicious-additional-filters</artifactId>
+                <groupId>io.kroxylicious.filter</groupId>
```

You've still got a stray ref to the `io.kroxylicious.filter` group.

```suggestion
                <groupId>io.kroxylicious</groupId>
```